Add the verification of drivers at troubleshooting installing-and-updating-drivers.md (#171)
Microsoft-community/Community-website: wiki/installing-and-updating-drivers.md

## Troubleshooting 🔧
### If you can access Windows:
#### You can verify if all of your drivers are signed.
1. Press <kbd>Win</kbd> + <kbd>R</kbd> and write `sigverif` and press <kbd>enter</kbd>.

> 2. It will pop a window to verify and you press the <kbd>Start</kbd> button.

Reviewer: Maybe change it to "A window will pop up. Press the <kbd>Start</kbd> button to verify."?

Reviewer 2: ```suggestion
2. A window will pop up. Press the `Start` button to verify driver signatures.
```

## Troubleshooting 🔧
### If you can access Windows:

> 1.1 You can verify if all of your drivers are signed, clicking <kbd>Win</kbd> + <kbd>R/kbd> and write <kbd>sigvetif</kbd> and press <kbd>enter</kbd>.

Reviewer 3: fixing the netlify deploy error

```suggestion
1.1 You can verify if all of your drivers are signed, clicking <kbd>Win</kbd> + <kbd>R</kbd> and write <kbd>sigvetif</kbd> and press <kbd>enter</kbd>.
```

## Troubleshooting 🔧
### If you can access Windows:
#### 1. You can verify if all of your drivers are signed.

> Press <kbd>Win</kbd> + <kbd>R</kbd> and write <kbd>sigvetif</kbd> and press <kbd>enter</kbd>.

Reviewer 2: ```suggestion
Press <kbd>Win</kbd> + <kbd>R</kbd> and write <kbd>sigverif</kbd> and press <kbd>enter</kbd>.
```

## Troubleshooting 🔧
### If you can access Windows:
#### Verifying Driver Signatures
Verifying driver signatures ensures you are installing secure, untampered and correct drivers issued by your hardware manufacturer.

> 1. Press <kbd>Win</kbd> + <kbd>R</kbd>, then type in `sigverif` and press <kbd>enter</kbd>.

Reviewer 4: Apologies for being picky
```suggestion
1. Press <kbd>Win</kbd> + <kbd>R</kbd>, then type in `sigverif` and press <kbd>Enter</kbd>.
```

Author: I want the dessert to be as good as it can be @waitwhatActo

## Troubleshooting 🔧
### If you can access Windows:
#### Verifying Driver Signatures

> ::: Why you should do that?

Reviewer 4: I don't think that's how you're supposed to use tip notes

Reviewer 4: Personally, I think this should be in a paragraph of its own, not a tip note.

Author: OK, let me fix it now.

Verifying driver signatures ensures you are installing secure, untampered and correct drivers issued by your hardware manufacturer.
:::

> 1. Press <kbd>Win</kbd> + <kbd>R</kbd> and write `sigverif` and press <kbd>enter</kbd>.

Reviewer 4: ```suggestion
1. Press <kbd>Win</kbd> + <kbd>R</kbd>, then type in `sigverif` and press <kbd>enter</kbd>.
```

## Troubleshooting 🔧
### If you can access Windows:
#### Verifying Driver Signatures
::: Why you should do that

> Maybe you installed a non-official driver by error and can cause system issues.

Reviewer: Verifying driver signatures ensures you are installing secure, untampered and correct drivers issued by your hardware manufacturer.

Author: Fixed, thanks.
